Cargo.toml

```diff
@@ -2,3 +2,6 @@
 members = [
   "pkgs/cargo-messages"
 ]
+exclude = [
+  "test/integration/sniff-bytes"
```

Comment:
This is needed so the root cargo namespace doesn't get confused by the Rust code inside the integration tests.